Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -412,6 +412,12 @@ theorem isEquiv_of_map_strictMono [LinearOrderedCommMonoidWithZero Γ₀]
     (H : StrictMono f) : IsEquiv (v.map f H.monotone) v := fun _x _y =>
   ⟨H.le_iff_le.mp, fun h => H.monotone h⟩
 
+theorem isEquiv_iff_val_lt_val [LinearOrderedCommGroupWithZero Γ₀]
+    [LinearOrderedCommGroupWithZero Γ'₀] (v : Valuation K Γ₀) (v' : Valuation K Γ'₀) :
```

Comment:
```suggestion
    [LinearOrderedCommGroupWithZero Γ'₀] {v : Valuation K Γ₀} {v' : Valuation K Γ'₀} :
```

Reply:
I use `(v v')` here to be aligned with `isEquiv_of_val_le_one` and other lemmas. Do you think changing all `(v v')` into `{v v'}` is a good idea? I will make the change if you think this is a good idea.

Reply:
I'll open a new PR to change parenthesis in similar lemmas.